libs/ui/src/lib/metadata-card/metadata-card.component.stories.ts

```diff
@@ -0,0 +1,78 @@
+import { Meta, StoryObj } from '@storybook/angular';
+import notes from './metadata-card.component.md';
+import { MetadataCardComponent } from './metadata-card.component';
+
+export default {
+  title: 'Angular/Metadata Card',
+  component: MetadataCardComponent,
+  parameters: {
+    docs: {
+      description: {
+        component: notes,
+      },
+    },
+  },
+  argTypes: {
+    severity: {
+      name: 'Severity style',
+      options: ['info', 'success', 'warning', 'error'],
```

Comment:
No need in explicitly listing the options here - union types should be filled automatically by Storybook itself (with some rare exceptions).